lib/Optimizer/GraphOptimizerPipeline/Pipeline.cpp

```diff
@@ -110,6 +110,11 @@ FunctionPassPipeline glow::createDefaultGraphOptimizationPassPipeline() {
        ConvergenceMode::OnePass,
        {CompilationMode::Infer}},
 
+      // Fold MatMul + Add into FullyConnected.
```

Comment:
This better fits in `createDefaultFoldPassPipeline()` below -- otherwise if a backend does some lowering to create this MatMul -> Add pattern, this optimization would undo it. Fold only happens once, at the start of the optimization pipeline.

Reply:
I am not yet familiar with the pipeline organization so I followed your advice:)
Done.

Reply:
The Clang format job fails for some PyTorch files. Not sure what is going on with the CI ...

Reply:
Not your fault, another PR broke it, will fix separately. Thanks for fixing!